Mutagen.Bethesda.Analyzers.Testing/TestAnalyzer.cs

```diff
@@ -16,7 +18,10 @@ public void Analyze(ContextualAnalyzerParams param)
     {
         foreach (var topic in Topics)
         {
-            param.AddTopic(topic.Format());
+            param.AddTopic(
+                ModKey.Null,
+                new MajorRecordIdentifier { FormKey = FormKey.Null },
```

Comment:
I think we can re-add the `ContextualAnalyzerParams.AddTopic` alternative without the ModKey/Record parameters, so that calls like this don't need to "fake it"?

I guess that'd make it a little easier for contextual analyzer devs to avoid passing in the desired information

Reply:
The idea was to make it mandatory to pass in all the information. Otherwise it might be easy to forget it and to just use the other variant where you don't have to pass this info.
For the test I think it might be fine to fake it, or have a test-only extension method that allows this workaround?